src/renderer/components/_molecules/data-types/structure/table/elements/array-modal.tsx

```
@@ -0,0 +1,284 @@
+          <div
+            aria-label='Array type table container'
+            className='flex h-fit w-full flex-col *:border-x *:border-b *:border-neutral-300 [&>*:first-child]:rounded-t-lg [&>*:first-child]:border-t [&>*:last-child]:rounded-b-lg'
+          >
+            {dimensions.map((value, index) => (
+              <ArrayDimensionsInput
+                id={index.toString()}
+                initialValue={value}
+                selectedInput={selectedInput}
+                handleInputClick={handleInputClick}
+                handleUpdateDimension={handleUpdateDimension}
+              />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)